### ISSUE TITLE
1.6.4 - Print PNG, mapfish print v2

### ISSUE THREAD

**Issue by kalbermattenm** [camptocamp/c2cgeoportal#1683]
This one is quite obscure for me...

When trying to print a PNG file, then I get:
```
2015-10-02 11:17:11,476 [http-bio-8080-exec-5] INFO  put.NativeProcessOutputFactory - Executing process: [convert, -density, 254, -append, D:\Applications\ApacheTomcat\temp\mapfishprint5058496617940523542.pdf, -bordercolor, #FFFFFF, -border, 0, -alpha, off, D:\Applications\ApacheTomcat\temp\mapfishprint6016056180589975706.png]
2015-10-02 11:17:11,492 [http-bio-8080-exec-5] ERROR put.NativeProcessOutputFactory - Invalid Parameter - 254
2015-10-02 11:17:11,492 [http-bio-8080-exec-5] ERROR put.NativeProcessOutputFactory - convert failed to create image from pdf.  Exit code was 4
```
It produces an empty image (0 bytes).

Can be reproduced here: http://sitn.ne.ch/prepublication16/theme/main

I think this has something to do with ImageMagik, but I cannot get what... Any hint ?

**Comment by kalbermattenm** [comment]
OK... It looks like the `imagemagick-mapfish-spring-application-context-override.xml` is not taken into account when building the print. (I need it to be taken into account, because it specifies the installation path of ImageMagick on my beautiful Windows servers...).

If I copy-paste this file into the Tomcat webapps folder, then it works...

@sbrunner, do you think that we missed something in the makefile print section in order to add this file to the resulting war ?

**Comment by sbrunner** [comment]
Just for test if you set in your Makefile this:
`PRINT_INPUT_FIND = *.tif *.bmp *.jpg *.jpeg *.gif *.png *.pdf *.xml`
Is it working ?

**Comment by kalbermattenm** [comment]
Yes... This is working... Thanks a lot !

Is that a change that I should apply in the sitn_c2cgeoportal project or is that something that should be implemented in c2cgeoportal ?

**Comment by sbrunner** [comment]
in c2cgeoportal :-)

**Comment by sbrunner** [comment]
I do the pullrequest, you can backport it directly in your the `CONST_Makfile` in your project :-)

**Comment by kalbermattenm** [comment]
OK, thanks a lot !

**Comment by kalbermattenm** [comment]
Fixed by #1688, closing
